package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ideditor/nsi-collector",
-  "version": "1.0.20230416",
+  "version": "1.0.20230424",

```

Comment:
This is best done during the release process itself instead of these PRs

Reply:
I am referring to your previous commit and trying to imitate this release process, because I noticed that you didn‘t collect from latest planet data for a few days ago (maybe because of too busy?), so I wanted to keep it up to date and ran the scripts locally. 

https://github.com/ideditor/nsi-collector/commit/fb06cca997bf46f2211fed008631b214ee67db20

For process and PR, created these PRs (and #12) because they couldn't be directly pushed, if you it is  unnecessary for me, I won't continue to run it every week.

Reply:
I know this project is kind of painful to work with because of its dependence on `node-osmium`.
Would it be useful to have some more people who can make releases?  or automate more of it (#3). 
I admit I haven't looked at it in a while, but maybe there are ways to make it more frictionless?